.github/workflows/validate.yml

```diff
@@ -41,11 +41,14 @@ jobs:
       - name: Install Dependencies
         run: pnpm install --frozen-lockfile
 
+      - name: Set Puppeteer Cache Directory
+        run: echo "puppeteer_cache_dir=$HOME/.cache/puppeteer" >> "$GITHUB_ENV"
+
       - name: Cache Puppeteer
         id: puppeteer-cache
         uses: actions/cache@v3
         with:
-          path: /home/runner/.cache/puppeteer
+          path: ${{ env.puppeteer_cache_dir }}
```

Comment:
For this sort of thing, I prefer to have the output coupled to the step, like [here](https://github.com/seek-oss/crackle/blob/f6994aaa4d024adb8b515b4176d4307f8c93c3ae/.github/workflows/ci.yml#L122-L125). That way, there's no confusion as to where the value comes from.

Reply:
https://github.com/seek-oss/sku/pull/841/commits/d5716051f4195f665641057620947e75ab3f8238